app/routes/docs/history.mdx

```diff
@@ -0,0 +1,48 @@
+
+## Gamma-ray Burst Coordinates Network
+
+New instruments were added to the network, including the Rossi X-ray timing Explorer ([RXTE](https://heasarc.gsfc.nasa.gov/docs/xte/XTE.html)) Proportional Counter Array ([PCA](https://heasarc.gsfc.nasa.gov/docs/xte/PCA.html)) and the Array of Low Energy X-Ray Imaging Sensors ([ALEXIS](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/alexis.html)) extreme ultraviolet transient mission. GCN Notices from the CGRO Imaging Compton Telescope ([COMPTEL](https://heasarc.gsfc.nasa.gov/docs/cgro/comptel/)) were added in November 1997. In recognition of the broader usage, the system was renamed the Gamma-ray burst Coordinates Network (GCN) in 1997.
+
+An early, important discovery came from the transmission of the [BeppoSAX localization](https://gcn.nasa.gov/circulars/199) of GRB 990123 by the GCN system. This GRB location automatically steered the robotic telescope [ROTSE-I](https://gcn.nasa.gov/circulars/205) to GRB 990123 only 22 seconds after the start of the GRB, leading to the first detection of an optical flash during the prompt phase of a GRB.
+
+[GCN Circulars](/circulars) -- the curated observation reports were introduced in 1997, and has led to >35k circulars as of mid-2023. These citable, though non-peer reviewed, reports appear automatically in ADS and are regularly cited in the astronomical literature.
+
+## Gamma-ray Coordinates Network
+
+As GCN grew in its utilization beyond the GRB community for other types of high-energy astronomical transients (e.g. soft gamma-ray repeaters, tidal disruption events), it was renamed once again to the Gamma-ray Coordinates Network (also GCN) in ~2009.
+
+GCN continued to play a seminar role in enabling astrophysical discovery with the distribution of GCN Notices from the advanced gravitational wave network and astrophysical neutrinos from IceCube starting in ~2015. This included a completely parallel GCN Notices and Circulars system while gravitational alerts were distributed only to teams with MOUs with the LIGO Virgo Consortium, including the [first detection of a binary neutron star merger](https://gcn.nasa.gov/circulars?page=2&query=G298048) and its subsequent multiwavelength counterparts.
+
+The principle visionary and tireless developer of GCN over its first 30 years was Scott Barthelmy (NASA/GSFC) with significant contributions from Teresa Sheets, Craig Markwardt and Tom McGlynn, with [many others](https://gcn.gsfc.nasa.gov/people.html) making significant contributions to design and operation.
```

Comment:
"Visionary" is an accurate word for Scott's contribution, but it may come across as self-aggrandizing given that this copy is on our own web site. Is there another way that we can say this?

Reply:
We could change "principle visionary" to "principal investigator".  It's more of a technical term, but makes it clear that he pushed this effort along for all of those years.